checker/checkInductive.ml

```diff
@@ -143,7 +143,7 @@ let check_inductive env mind mb =
         mind_private; mind_typing_flags; }
     =
     (* Locally set the oracle for further typechecking *)
-    let env = Environ.set_oracle env mb.mind_typing_flags.conv_oracle in
+    let env = Environ.set_typing_flags mb.mind_typing_flags env in
```

Comment:
If -indices-matter is globally set we should either reject unsetting it here or list inductives which unset it.

Reply:
Done. Thanks